src/main/kotlin/io/csbroker/apiserver/service/common/NotificationServiceImpl.kt

```diff
@@ -69,4 +70,16 @@ class NotificationServiceImpl(
 
         return notificationRepository.countUnReadByUserId(findUser.id!!)
     }
+
+    @Transactional
+    override fun deleteNotifications(email: String, ids: List<Long>) {
+        val findUser = userRepository.findByEmail(email)
+            ?: throw EntityNotFoundException("$email 에 해당하는 유저를 찾을 수 없습니다.")
+
```

Comment:
```suggestion
        if (findUser.notifications.map { it.id }.toSet().containsAll(ids)) {
            throw UnAuthorizedException(ErrorCode.FORBIDDEN, "본인의 알림만 삭제할 수 있습니다.")
        }
```
의도하지 않은 버그를 방지하기 위해 본인인지 확인하는 로직이 있으면 좋지 않을까요??!

Reply:
@ekzm8523 잉 밑에서 지울 때, user 정보 기반으로 찾아서 굳이 검증이 필요한가용..

Reply:
테스트는 해보시죠!

Reply:
앗.. 난시가..